starter_code/src/movies.js

```diff
@@ -1,19 +1,60 @@
 /* eslint no-restricted-globals: 'off' */
 
-// Iteration 1: All rates average - Get the average of all rates with 2 decimals 
+// Iteration 1: All rates average - Get the average of all rates with 2 decimals
+function ratesAverage(movies) {
+  let totalSum = movies.reduce(function(accumulator, value) {
+    return accumulator + parseInt(value.rate, 10);
+  }, 0);
+  return totalSum / movies.length;
+}
 
- 
 // Iteration 2: Drama movies - Get the average of Drama Movies
 
+function dramaMoviesRate(movies) {
+  let dramaMovies = movies.filter(function(movie) {
+    return movie.genre.includes("Drama");
+  });
+  if (dramaMovies.length == 0) {
+    return 0;
+  } else {
+    let totalDramaRates = dramaMovies.reduce(function(accumulator, dramaMovie) {
+      return accumulator + dramaMovie.rate * 1;
+    }, 0);
+    return Math.round((totalDramaRates / dramaMovies.length) * 100) / 100;
```

Comment:
This can be rewritten as:

  return Number((totalDramaRates / dramaMovies.length).toFixed(1));

(Number means you want to turn the result into a number (not a string)). 
(toFixed(1) means you want to only return one decimal place)